Feature: created buttons component create buttons and OR divider

SufyanCS opened the pull request:
Created authentication Buttons with OR Divider component and make it responsive for Desktop, Tablet and Mobile.
Components struture and Desktop version:
![ButtonAuth-Desktop](https://user-images.githubusercontent.com/119085953/229333068-48907b46-adf5-481b-ae66-f61c69df27f8.png)
Tablet and obile version:
![ButtonAuth-Tablet-Mobile](https://user-images.githubusercontent.com/119085953/229333177-f22a697a-f906-4986-be52-7e0f938098ff.png)

SufyanCS committed:
feat(created authentication accounts buttons component): create buttons and OR divider

Created authentication Buttons with OR Divider component and make it responsive for Desktop, Tablet
and Mobile.

adeeb-samad reviewed (commented):
Reviewed

adeeb-samad reviewed (approved):
Done

iiAbady commented on src/components/ButtonAuth.jsx:
```diff
@@ -0,0 +1,29 @@
+                className={`w-full h-14 px-6 transition ${styles.ButtonAuth}`}
+            >
+                <div className='relative flex items-center space-x-4 justify-center'>
+                    <Image
+                        src={props.icon}
+                        alt={props.alt}
+                        width={props.width}
+                        height={props.height}
+                    />
```
@SufyanCS Great work! I have a little suggestion here to destruct these values and use them instead of typing `props` every time.
```javascript
const { icon, alt, width, height } = props // at the start of the component.
```

SufyanCS committed:
Update ButtonAuth.jsx component

the props object is being destructured into separate variables icon, alt, width, and height to access them easily.
